share/empty.go

```diff
@@ -28,18 +28,11 @@ func EmptyEDS() *rsmt2d.ExtendedDataSquare {
 	return emptyBlockEDS
 }
 
-// EmptyBlockShares returns the shares of the empty block.
-func EmptyBlockShares() []libshare.Share {
-	initEmpty()
-	return emptyBlockShares
-}
-
```

Comment:
I would keep this one. You never know

Reply:
Shares can be obtained from chained call EmptyEDS.FlattenedODS(), so there is not much value in additional global stored shares

Reply:
But its ods, not eds

Reply:
The idea is similar, there is `Flattened()` for all eds shares